src/components/panel/examples/announcement/index.njk

```diff
@@ -0,0 +1,18 @@
+---
+"theme": census
+"fullWidth": true
+---
+{% from "components/panel/_macro.njk" import onsPanel %}
+
+{% call onsPanel({
+        "type": 'announcement'
+    })
+%}
+
+<p class="u-mb-no">National lockdown: stay at home</p>
+
+<div class="u-fs-r">
+    <p>Coronavirus (COVID-19) remains a serious threat across the country. <a href="#0">Find out more</a></p>
+</div>
```

Comment:
This will probably have to use WYSIWYG in Craft cms, so we wouldn't want to have to manually add html markup.  I'd suggest remove the bold css from the announcement style type so it can be added (if required) using `strong` tags in the editor.  

The first line would probably have to be a heading? So might want to control font size of `h2 ` in announcement type with css

Reply:
When we know what the content is we can revisit this component.